src/main/java/reposense/util/TimeUtil.java

```diff
@@ -0,0 +1,29 @@
+package reposense.util;
+
+/**
+ * Contains time related functionalities.
+ */
+public class TimeUtil {
+    private static final String MESSAGE_ELAPSED_TIME =
+            "Elapsed processing time: %d hour(s) %d minute(s) %.2f second(s)";
```

Comment:
Not sure if I am too mean or not. I suggest not to have `hour` / `minute` displayed if the build does not take an `hour` / `minute` to finish. 

However, this is just left as a suggestion. May go ahead after seniors also agree with this.

Reply:
I personally agree with this also

Reply:
@fzdy1914 can I continue with making this change?

Reply:
Sure, if you also agree with this suggestion.